requirements-frozen.txt

```diff
@@ -0,0 +1,17 @@
+antlib==1.1b0
+Click==7.0
```

Comment:
I think it would be good to use the opportunity to bump all dependencies to latest available. The same exercise could be repeated for every release to stay up to date.

Reply:
Both `antlib 1.1b0` and `Click 7.0` are the latest releases available on PyPI. Which are you pointing at?

Reply:
I was thinking all the dependencies in general, not pointing to any in particular. If all are up to date then all is good.